services/viva-ms/helpers/createRecurringCaseTemplateData.js

```diff
@@ -59,6 +59,20 @@ export function createPersonsObject(persons, answers) {
   return applicantPersons;
 }
 
+function createNotesObject(answers) {
+  const notes = [];
+  const filteredAnswers = formHelpers.filterByFieldIdIncludes(answers, 'otherMessage');
+  if (filteredAnswers.length) {
+    const [noteAnswer] = filteredAnswers;
+    const note = {
+      title: 'Meddelande från sökande',
+      text: noteAnswer.value,
+    };
+    notes.push(note);
+  }
+  return notes;
+}
+
```

Comment:
Not sure if notes array will be populated with more than one item with this implementation?
Have a hard time to graps this :)

Reply:
If you look at the pdf design the uppermost section of the pdf contains two "rows", But i'm uncertain what content goes into the first "row".

The two type of notes:
Har något ändrat sedan senast
Meddelande från sökande

Reply:
So currently the array only consits of one object, but the rendering supports multiple objects for the "notes" section.